test/test_issue_scan.py

```diff
@@ -172,20 +172,40 @@ def test_scan_ghapi_amqp(self, mock_queue):
         self.assertEqual(channel.basic_publish.call_args_list[0][0][1], "public")
         request = json.loads(channel.basic_publish.call_args_list[0][0][2])
 
-        self.assertEqual(request, {
+        assert request == {
```

Comment:
TBH I'd prefer setting `self.maxDiff = None` here, as it pretty much breaks running with unittest (the failure output is useless). But no strong opinion.